code/modules/projectiles/projectile/bullets/pistol.dm

```diff
@@ -61,3 +61,9 @@
 	impact_light_intensity = 5
 	impact_light_range = 1
 	impact_light_color_override = LIGHT_COLOR_DIM_YELLOW
+
+/obj/projectile/bullet/fortynr
+	name = "40nr bullet"
+	damage = 20
+	stamina = 30
+	weak_against_armour = TRUE
```

Comment:
Is there any particular reason this has to be weak against armor while doing similar damage to a disabler and laser?

This isn't nonlethal enough for compliance, but isn't lethal enough where it is a better alternative to a laser. The most meaningful use I could see for it is killing specifically spiders due to their stamina damage vulnerability.

Reply:
Yes, blunt bullets. They have big stopping power, but don't pierce armor